services/storage/src/simcore_service_storage/application.py

```diff
@@ -72,6 +72,11 @@ def create(settings: Settings) -> web.Application:
 
     app.middlewares.append(dsm_exception_handler)
 
+    if settings.STORAGE_PROFILING_MIDDLEWARE_ENABLED:
+        from servicelib.aiohttp.profiler_middleware import create_profiling_middleware
```

Comment:
Q: why is this import here? is it really needed?

Reply:
Yes, I just did because it made sense back when I originally added this to the api-server. I have changed.